test/unit/data.readers/process.space.tree.test.ts

```
@@ -0,0 +1,242 @@
+        id: 'space-1',
+        collaboration: { calloutsSet: { callouts: [] } },
+      },
+    ];
+
+    const result = await processSpaceTree(spaces as any, mockAlkemioClient);
+
+    expect(result).toHaveLength(1);
+  });
+
+  it('should process multiple root-level spaces', async () => {
+    let callCount = 0;
```

Biome (2.4.9)

[error] This variable callCount is unused.

(lint/correctness/noUnusedVariables)